python/cloud-run-django-hello-world/.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/cloud-run-django-hello-world.iml" filepath="$PROJECT_DIR$/.idea/django-hello-world.iml" />
```

Comment:
Just verifying - is the dollar sign at the end of the variable "by design"?

Reply:
I just copied this from https://github.com/GoogleCloudPlatform/cloud-code-samples/blob/master/golang/go-hello-world/.idea/modules.xml. I'm unfamiliar with IntelliJ, so I don't know whether this is right or wrong.